gnocchi/incoming/__init__.py

```diff
@@ -166,11 +220,18 @@ def process_measure_for_metrics(metric_id):
     def has_unprocessed(metric_id):
         raise exceptions.NotImplementedError
 
+    def _get_sack_name(self, number):
```

Comment:
maybe drop leading '_' as you call this beyond this module.

Reply:
I don't call it outside, I call it in subclasses only. It's not meant to be public.